### PR TITLE
Issue 103

### PR DESCRIPTION
**Pull request by BasThomas**

### CODE REVIEW

**Comment by BasThomas** [comment]
This is done, would love some feedback. :)

File: _drafts/2018-02-22-issue-103.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Joe Groff [wrote](http://duriansoftware.com/joe/Optimizing-global-constant-data-structures-using-relative-references.html) a blog post on optimizing global constant data structures.
+
+Paul Hudson [gives on overview](https://www.hackingwithswift.com/articles/55/how-to-use-dynamic-member-lookup-in-swift) of what [SE-0195](https://github.com/apple/swift-evolution/blob/master/proposals/0195-dynamic-member-lookup.md): *Introduce User-defined "Dynamic Member Lookup" Types* brings to the table to make Swift more dynamic.
+
+Keith Harrison [explains](https://useyourloaf.com/blog/replacing-flatmap-with-compactmap/) why one of `flatMap`'s implementations is now called `compactMap`, as per [SE-0187](https://github.com/apple/swift-evolution/blob/master/proposals/0187-introduce-filtermap.md): *Introduce `Sequence.compactMap(_:)`*.
+
+Erik Eckstein https://swift.org/blog/osize/
+
+Ian McDowell https://twitter.com/lapcatsoftware/status/962347634408280065
```
Comment (jessesquires) [inline _drafts/2018-02-22-issue-103.md]: Remove these two above ??

Comment (BasThomas) [reply _drafts/2018-02-22-issue-103.md]: Oops, good catch

**Comment by BasThomas** [comment]
⚠️ Moved the draft to `_posts`, so merging will publish :)

**Comment by jessesquires** [comment]
Looks great @BasThomas !

**Comment by BasThomas** [comment]
Thanks! :)

**Pull request merged**